client/multi-currency/blocks/currency-switcher.js

```diff
@@ -0,0 +1,318 @@
+			setAttributes( {
+				fontSize: newFontSize === undefined ? 11 : newFontSize,
+			} );
+		};
+
+		const onChangeFontLineHeight = ( newFontLineHeight ) => {
+			setAttributes( {
+				fontLineHeight:
+					newFontLineHeight === undefined
+						? 'normal'
+						: newFontLineHeight,
+			} );
+		};
+
+		const onChangeFontColor = ( newFontColor ) => {
+			setAttributes( {
+				fontColor:
+					newFontColor === undefined ? '#000000' : newFontColor,
+			} );
+		};
+
+		const onChangeBorder = ( newBorder ) => {
+			setAttributes( {
+				border: newBorder === undefined ? false : newBorder,
+			} );
+		};
+
+		const onChangeBorderRadius = ( newBorderRadius ) => {
+			setAttributes( {
+				borderRadius:
+					newBorderRadius === undefined ? 3 : newBorderRadius,
+			} );
+		};
+
+		const onChangeBorderColor = ( newBorderColor ) => {
+			setAttributes( {
+				borderColor:
+					newBorderColor === undefined ? '#000000' : newBorderColor,
+			} );
+		};
+
+		const onChangeBackgroundColor = ( newBackgroundColor ) => {
+			setAttributes( {
+				backgroundColor:
+					newBackgroundColor === undefined
+						? 'transparent'
+						: newBackgroundColor,
+			} );
+		};
+
+		return (
+			<div { ...blockProps }>
+				<InspectorControls key="setting">
+					<PanelBody
+						title={ __(
+							'Multi-Currency Settings',
+							'woocommerce-payments'
+						) }
+					>
+						<a
+							href={
+								'/wp-admin/admin.php?page=wc-settings&tab=wcpay_multi_currency'
+							}
+							target="_blank"
+							rel="noreferrer"
+						>
+							Adjust and edit your Multi-Currency settings
+						</a>
+					</PanelBody>
+					<PanelBody title={ __( 'Layout', 'woocommerce-payments' ) }>
+						<CheckboxControl
+							label={ __(
+								'Display flags',
+								'woocommerce-payments'
+							) }
+							checked={ flag }
+							onChange={ onChangeFlag }
+						/>
+
+						<CheckboxControl
+							label={ __(
+								'Display currency symbols',
+								'woocommerce-payments'
+							) }
+							checked={ symbol }
+							onChange={ onChangeSymbol }
+						/>
+
+						<CheckboxControl
+							label={ __( 'Border', 'woocommerce-payments' ) }
+							checked={ border }
+							onChange={ onChangeBorder }
+						/>
+
+						<RangeControl
+							label={ __(
+								'Border radius',
+								'woocommerce-payments'
+							) }
+							value={ borderRadius }
+							onChange={ onChangeBorderRadius }
+							min={ 1 }
+							max={ 20 }
+						/>
+					</PanelBody>
+					<PanelBody
+						title={ __( 'Typography', 'woocommerce-payments' ) }
+					>
+						<RangeControl
+							label={ __( 'Size', 'woocommerce-payments' ) }
+							value={ fontSize }
+							onChange={ onChangeFontSize }
+							min={ 6 }
+							max={ 48 }
+						/>
+						<RangeControl
+							label={ __(
+								'Line height',
+								'woocommerce-payments'
+							) }
+							value={ fontLineHeight }
+							onChange={ onChangeFontLineHeight }
+							min={ 1 }
+							max={ 3 }
+							step={ 0.1 }
```

Comment:
Will it be better using pixels instead em's as the unit for the lineHeight? @ismaeldcom what do you think?

Reply:
I think em is a much more confusing unit than pixels for average users.

Reply:
I normally prefer to use `em` but looks like we tend to use `px` in our CSS customizations, and I think in this case it would be easier for the end user.

Reply:
It's used without units in the renderer side, so it acts like a multiplier of the font-size (100%, 110%, 120%..). We can keep it this way.